components/shared/AddressChecker.vue

```diff
@@ -120,6 +122,18 @@ const checkAddressByss58Format = (value: string, ss58: number) => {
 }
 
 const getAddressCheck = (value: string): AddressCheck => {
+  return execByVm({
```

Comment:
Type 'AddressCheck | undefined' is not assignable to type 'AddressCheck'.

Reply:
fixed

Reply:
fixed